classification2.Rmd

```diff
@@ -1012,7 +1050,7 @@ one over increasing predictor set sizes
 and another to check which predictor to add in each round (where you see `for (j in 1:length(names))` below).
 For each set of predictors to try, we construct a model formula,
 pass it into a `recipe`, build a `workflow` that tunes
-a KNN classifier using 5-fold cross-validation, 
+a $K$-NN classifier using 5-fold cross-validation, 
 and finally records the estimated accuracy.
 
 ```{r 06-fwdsel-2, warning = FALSE}
```

Comment:
This code block has a for loop in it. Tiffany said she would make a package for this, and we should definitely make sure to do that. Might want to ping her to do that and then edit this PR accordingly.

Reply:
#242